elliott/elliottlib/cli/find_bugs_sweep_cli.py

```diff
@@ -353,7 +353,7 @@ def categorize_bugs_by_type(bugs: List[Bug], advisory_id_map: Dict[str, int],
 
         for bug in tracker_bugs:
             package_name = bug.whiteboard_component
-            if package_name == "microshift" and len(packages) == 0:
+            if kind == "microshift" and package_name == "microshift" and len(packages) == 0:
```

Comment:
if advisory is of kind `microshift`, will the package always `microshift`? Is there a redundancy here?